docs/conf.py

```diff
@@ -62,7 +62,7 @@
 
 # General information about the project.
 project = 'sparse'
-copyright = '2017, Sparse Developers'
+copyright = '2018, Sparse developers'
```

Comment:
Capitalization changed to better match SciPy and NumPy, and year changed to 2018.